scripts/miniapps.py

```diff
@@ -526,6 +528,22 @@ def evp(
         env += " OMP_NUM_THREADS=1"
         app = f"{build_dir}/miniapp/miniapp_eigensolver"
         opts = f"--matrix-size {m_sz} --block-size {mb_sz} {band_flag} --grid-rows {grid_rows} --grid-cols {grid_cols} --nruns {nruns} {extra_flags}"
+    elif lib == "scalapack":
+        env += f" OMP_NUM_THREADS={cores_per_rank}"
+        app = f"{build_dir}/miniapp_evp_scalapack"
+        opts = f"{m_sz} {mb_sz} {grid_rows} {grid_cols} {nruns}"
+    elif lib == "elpa1":
+        if system["GPU"]:
+            env += f" env 'ELPA_DEFAULT_nvidia-gpu=1'"
+        env += f" ELPA_DEFAULT_omp_threads={cores_per_rank} OMP_NUM_THREADS={cores_per_rank}"
+        app = f"{build_dir}/miniapp_evp_elpa"
+        opts = f"{m_sz} {mb_sz} {grid_rows} {grid_cols} {nruns} 1"
+    elif lib == "elpa2":
+        if system["GPU"]:
+            env += f" env 'ELPA_DEFAULT_nvidia-gpu=1'"
+        env += f" ELPA_DEFAULT_omp_threads={cores_per_rank} OMP_NUM_THREADS={cores_per_rank}"
+        app = f"{build_dir}/miniapp_evp_elpa"
+        opts = f"{m_sz} {mb_sz} {grid_rows} {grid_cols} {nruns} 2"
```

Comment:
Maybe? I'm happy with the current code as well...
```suggestion
    elif lib.startswith("elpa") == "elpa1":
        if system["GPU"]:
            env += f" env 'ELPA_DEFAULT_nvidia-gpu=1'"
        env += f" ELPA_DEFAULT_omp_threads={cores_per_rank} OMP_NUM_THREADS={cores_per_rank}"
        app = f"{build_dir}/miniapp_evp_elpa"
        opts = f"{m_sz} {mb_sz} {grid_rows} {grid_cols} {nruns}"
        if lib == "elpa1":
            opts += " 1"
        elif lib == "elpa2":
            opts += " 2"
        else
            # maybe assert?
```

Reply:
@msimberg @albestro Is https://github.com/eth-cscs/DLA-Future/pull/787/commits/03917fb97f66fec1d9734159f8c4e251f0c895c0 ok?